internal/db/schema/migrations/postgres/3/01_oidc.up.sql

```diff
@@ -0,0 +1,28 @@
+begin;
+
+
+-- auth_oidc_scope entries are the optional scopes for a specific oidc auth
+-- method.  There can be 0 or more for each parent oidc auth method.  If an auth
+-- method has any scopes, they will be added to provider requests along with the
+-- default of "openid". 
+create table auth_oidc_scope (
+  create_time wt_timestamp,
```

Comment:
Should there be a trigger to handle the create_time?

Reply:
ty!